geowebcache/core/src/test/java/org/geowebcache/TestHelpers.java

```diff
@@ -30,7 +31,8 @@
  */
 public class TestHelpers {
 
-    static GridSetBroker gridSetBroker = new GridSetBroker(false, false);
+    static GridSetBroker gridSetBroker =
+            new GridSetBroker(Collections.singletonList(new DefaultGridsets(false, false)));
```

Comment:
This is such a common snippet was it inlined? It may be worth having a new GridSetBroker() constructor that is configured with a singleton list as illustrated above.

Reply:
The constructor was deprecated, with instructions to do exactly the above:
https://github.com/GeoWebCache/geowebcache/blob/112060ad43708c030085fbbb147e6437a5a7e558/geowebcache/core/src/main/java/org/geowebcache/grid/GridSetBroker.java#L56

If each dev coming in tries to apply his on personal preference we'd just get chaos, I've tried not to question too much deprecations, unless they were evidently broken, .
That said, to be honest, I did not like this one, and a few others in GWC, that made for pretty tangled up replacement code especially in the GeoServer.